Basic-Video-Chat/.gitignore

```diff
@@ -0,0 +1,16 @@
+# intellij
+*.iml
+
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+app/build
+
+.settings/
+app/jniLibs/
+app/libs/opentok-android-sdk.jar
```

Comment:
Do we need to indicate that? IMHO, it doesn't exist for some versions ago and it could confuse. wdyt?